bundles/org.openhab.binding.omnikinverter/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -66,38 +77,92 @@
 		<description>The instantaneous power generation for output 3</description>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
+	<channel-type id="voltageAC1" advanced="true">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage AC1</label>
+		<description>The voltage for output 1</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="voltageAC2" advanced="true">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage AC2</label>
+		<description>The voltage for output 2</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="voltageAC3" advanced="true">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage AC3</label>
+		<description>The voltage for output 3</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
```

Comment:
There is no need to define three different channel types. If only label and description differs, you can define just one type and reuse that for all three channels. In that case you can override label and description directly in the channel definition - like this:
```xml
<channel id="voltageAC1" typeId="voltage">
    <label>Voltage AC1</label>
    <description>The voltage for output 1</description>
</channel>
```

Perhaps you can even just use one of the new system channel types - see openhab/openhab-core#3079.

```xml
<channel id="voltageAC1" typeId="system.electric-voltage">
    <label>Voltage AC1</label>
    <description>The voltage for output 1</description>
</channel>
```

Reply:
See #108ec4c for the changes